.env.development

```diff
@@ -1,11 +1,13 @@
 # App settings
-export DISABLE_DASHBOARD_REFRESH=true
+DISABLE_DASHBOARD_REFRESH=true
 
 # Database settings
 DEFENCE_SOLICITOR_DATABASE_HOST=localhost
 DEFENCE_SOLICITOR_DATABASE_PASSWORD=""
 DEFENCE_SOLICITOR_DATABASE_USERNAME="postgres"
 
+FORCE_SSL=false
```

Comment:
Question for everyone - given that the force_ssl option is only set in production.rb, do you think this is overkill?